### ISSUE TITLE
Error in callback after 5.3.2 update

### ISSUE THREAD

**Issue by mikkeldamm** [ePay/woocommerce#55]
Hi,

After the recent update to 5.3.2, then it seems that the change https://github.com/ePay/woocommerce/commit/46d7546024c08467d03b57c9a66ff1a2d85e5b65 cause en error because the function `getWPMLOrderLanguage` requires a WC_Order but in the latest change its the Order ID that is passed to the function.

I have tested by reverting the change in https://github.com/ePay/woocommerce/commit/46d7546024c08467d03b57c9a66ff1a2d85e5b65 locally and then it works again when the callback is being called.

Here is a log of the error:
```
CRITICAL Uncaught TypeError: Argument 1 passed to Bambora_Online_Classic_Helper::getWPMLOrderLanguage() must be an instance of WC_Order, int given, called in /nas/content/live/SITE/wp-content/plugins/bambora-online-classic/lib/bambora-online-classic-helper.php on line 295 and defined in /nas/content/live/SITE/wp-content/plugins/bambora-online-classic/lib/bambora-online-classic-helper.php:323
Stack trace:
#0 /nas/content/live/SITE/wp-content/plugins/bambora-online-classic/lib/bambora-online-classic-helper.php(295): Bambora_Online_Classic_Helper::getWPMLOrderLanguage(11680)
#1 /nas/content/live/SITE/wp-content/plugins/bambora-online-classic/bambora-online-classic.php(761): Bambora_Online_Classic_Helper::validate_bambora_online_classic_callback_params(Array, 'ABCD', Object(Automattic\WooCommerce\Admin\Overrides\Order), '')
#2 /nas/content/live/SITE/wp-includes/class-wp-hook.php(307): Bambora_Online_Classic->bambora_online_classic_callback('')
```

**Comment by sessan** [comment]
@mikkeldamm Hello! I will have a look at this asap. Thanks for the report!

**Comment by sessan** [comment]
Thanks @mikkeldamm  for noticing this issue. It is now resolved in the new 5.3.3 version.
